frontend/src/lib/components/CreateTable.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+    import { goto } from '$app/navigation';
+    import type {TableResponse} from "./CreateTable.models";
+
+    let username = '';
+
+    const createBoard = (): void => {
+        if (!username || !username.trim().length) {
+            return;
+        }
+        fetch(`http://localhost:8080/poker_api/v1/table`, {
```

Comment:
suggestion: extract it to a separate file . . .

Reply:
done